dcat/index.html

```diff
@@ -1872,6 +1876,83 @@ <h4>Property: was generated by</h4>
+            <a href="#Property:resource_access_rights">access rights</a>,
+            <a href="#Property:resource_conforms_to">conforms to</a>,
+            <a href="#Property:resource_contact_point">contact point</a>,
+            <a href="#Property:resource_creator">creator</a>,
+            <a href="#Property:resource_description">description</a>,
+            <a href="#Property:resource_has_policy">has policy</a>,
+            <a href="#Property:resource_identifier">identifier</a>,
+            <a href="#Property:resource_is_referenced_by">is referenced by</a>,
+            <a href="#Property:resource_keyword">keyword/tag</a>,
+            <a href="#Property:resource_landing_page">landing page</a>,
+            <a href="#Property:resource_license">license</a>,
+            <a href="#Property:resource_language">resource language</a>,
+            <a href="#Property:resource_relation">relation</a>,
+            <a href="#Property:resource_rights">rights</a>,
+            <a href="#Property:resource_qualified_relation">qualified relation</a>,
+            <a href="#Property:resource_publisher">publisher</a>,
+            <a href="#Property:resource_release_date">release date</a>,
+            <a href="#Property:resource_theme">theme/category</a>,
+            <a href="#Property:resource_title">title</a>,
+            <a href="#Property:resource_type">type/genre</a>,
+            <a href="#Property:resource_update_date">update/modification date</a>,
+            <a href="#Property:resource_qualified_attribution">qualified attribution</a>. 
+            <a href="#Property:dataset_frequency">frequency</a>,
+            <a href="#Property:dataset_spatial">spatial/geographic coverage</a>,
+            <a href="#Property:dataset_spatial_resolution">spatial resolution</a>,
+            <a href="#Property:dataset_temporal">temporal coverage</a>,
+            <a href="#Property:dataset_temporal_resolution">temporal resolution</a>,
+            <a href="#Property:dataset_was_generated_by">was generated by</a>.
+        </p>
+        <div class="issue" title="Any other properties to mention in this class ?" data-number="1308"></div>
+    
+        <table class="definition">
+            <thead><tr><th>RDF Class:</th><th><a href="http://www.w3.org/ns/dcat#DatasetSeries">dcat:DatasetSeries</a></th></tr></thead>
+            <tbody>
+            <tr><td class="prop">Definition:</td><td>A collection of datasets that are published separatelly, but share some common characteristics that groups them. <!--, and could also be made available as a single dataset.--></td></tr>
+            <tr><td class="prop">Sub-class of:</td><td><a href="#Class:Dataset"><code>dcat:Dataset</code></a></td></tr>
+            <tr><td class="prop">Usage note:</td><td> Dataset series can be also soft-typed via property <code>dct:type</code> as in the approach used in [[GeoDCAT-AP]], and adopted in [[DCAT-AP-IT]] and [[GeoDCAT-AP-IT]]).</td></tr>
```

Comment:
I think another Usage note would be helpful, giving common examples in a generic way. e.g. 
"Common scenarios for dataset series include: time series composed of periodically released subsets; map-series composed of items of the same type or theme but with differing spatial footprints."

Reply:
thanks, added the usage note as suggested